### PR TITLE
feat(cmd): install latest tagged image on `helm init`

### LINKED ISSUE #1117: feat: helm init should install released tiller
Currently running `helm init` installs the canary tiller image

### PR DESCRIPTION
**Pull request by adamreese**
closes: #1117

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/helm/1118)

### CODE REVIEW

File: cmd/helm/installer/install.go [hunk 1]
```diff
@@ -43,59 +41,91 @@ func Install(namespace, image string, verbose bool) error {
...
+	c, err := kc.Client()
+	if err != nil {
 		return err
 	}
 
-	if verbose {
-		fmt.Println(b.String())
+	ns := generateNamespace(namespace)
+	if _, err := c.Namespaces().Create(ns); err != nil {
+		if !errors.IsAlreadyExists(err) {
```
Comment (technosophos) [inline cmd/helm/installer/install.go]: Ooo... I didn't know about that errors helper.

File: cmd/helm/installer/install.go [hunk 2]
```diff
@@ -43,59 +41,91 @@ func Install(namespace, image string, verbose bool) error {
...
+	_, err = c.ReplicationControllers(namespace).Create(rc)
+	return err
+}
+
+func generateLabeles(labels map[string]string) map[string]string {
+	labels["app"] = "helm"
+	return labels
+}
+
+func generateReplicationController(image string) *api.ReplicationController {
```
Comment (technosophos) [inline cmd/helm/installer/install.go]: Why did we go from a Deployment to a ReplicationController?

Comment (adamreese) [inline cmd/helm/installer/install.go]: opps, this was an old branch I brought back to life, I will fix

File: cmd/helm/installer/install.go [hunk 3]
```diff
@@ -43,59 +41,91 @@ func Install(namespace, image string, verbose bool) error {
...
 	}
 
-	return kc.Create(namespace, &b)
+	rc := generateReplicationController(image)
+
+	_, err = c.ReplicationControllers(namespace).Create(rc)
+	return err
+}
+
+func generateLabeles(labels map[string]string) map[string]string {
```
Comment (technosophos) [inline cmd/helm/installer/install.go]: Are you trying to trick me with a spelling error?

File: cmd/helm/installer/install.go [hunk 4]
```diff
@@ -17,21 +17,19 @@ limitations under the License.
...
 	"k8s.io/helm/pkg/kube"
+	"k8s.io/helm/pkg/version"
 )
 
-// Install uses kubernetes client to install tiller
-//
-// Returns the string output received from the operation, and an error if the
-// command failed.
-//
-// If verbose is true, this will print the manifest to stdout.
```
Comment (technosophos) [inline cmd/helm/installer/install.go]: Why did the poor docstring get deleted?

**Pull request merged**